packages/design-system/src/styles/components/_StepList.scss

```diff
@@ -30,34 +16,34 @@ $step-button-height: 38px !default;
 .ds-c-step {
   color: $steplist__color;
   counter-increment: step-counter;
-  margin-bottom: $step-margin;
-  padding-bottom: $step-margin;
+  margin-bottom: $spacer-2;
+  padding-bottom: $spacer-2;
   position: relative;
 
   &::before {
-    border: $step-list-number-border-size solid $steplist-step__border-color--default;
+    border: 2px solid $steplist-step__border-color--default;
     border-radius: $radius-circle;
     box-sizing: border-box;
     color: $steplist-step__color;
     content: counter(step-counter);
     display: block;
-    font-size: $step-list-number-font-size-mobile;
-    height: $step-list-number-size-mobile;
-    left: calc(-1 * (#{$step-list-number-margin} + #{$step-list-number-size-mobile}));
-    line-height: $step-list-number-size-mobile - $step-list-number-border-size * 2;
+    font-size: $font-size-sm;
+    height: $spacer-3;
+    left: -30px;
```

Comment:
Not sure if this is the line, but looks like the number is not positioned in the same place it was

![step-list-change](https://user-images.githubusercontent.com/7595652/205172740-5f8a3568-7957-4621-8eff-c77c4effda31.gif)